### PR TITLE
Update Cryptocurrency Data

### PR DESCRIPTION
**Pull request by FedeRosaless**
https://user-images.githubusercontent.com/105088045/218114432-ea626b3e-21b8-4b19-bea8-8e9d0354c10a.mp4

### CODE REVIEW

File: src/store/actions.ts [hunk 1]
```diff
@@ -13,3 +13,18 @@ export const getCrypto = createAsyncThunk(
     }
   },
 );
+
+export const updateCrypto = createAsyncThunk(
+  'crypto/updateCrypto',
+  async (name: string[]) => {
+    let value: Crypto[] = [];
```
Comment (FedericoLeiva12) [inline src/store/actions.ts]: It should be const

File: src/store/actions.ts [hunk 2]
```diff
@@ -13,3 +13,18 @@ export const getCrypto = createAsyncThunk(
...
+    for (let i = 0; i < name.length; i++) {
+      let response = await fetch(
```
Comment (FedericoLeiva12) [inline src/store/actions.ts]: It should be const

**Comment by FedeRosaless** [comment]
https://user-images.githubusercontent.com/105088045/218137441-971ec39e-6270-4c89-8da5-6ab30fc705f5.mp4

File: src/screens/CryptoList/index.tsx [hunk 3]
```diff
@@ -1,18 +1,27 @@
...
 const CryptoList = () => {
+  const dispatch = useDispatch<AppDispatch>();
   const {navigate} = useNavigation();
   const {
     selectedCrypto: {cryptos},
   } = useSelector((state: RootState) => state);
 
+  useEffect(() => {
+    const time = setInterval(() => {
+      dispatch(updateCrypto(cryptos.map(c => c.name)));
```
Comment (FedericoLeiva12) [inline src/screens/CryptoList/index.tsx]: use destructuring
(cryptos.map(({name}) => name))

File: src/store/reducer/RootReducer.ts [hunk 4]
```diff
@@ -22,6 +22,9 @@ const cryptoSlice = createSlice({
       })
       .addCase(getCrypto.rejected, state => {
         state.loading = false;
+      })
+      .addCase(updateCrypto.fulfilled, (state, action) => {
```
Comment (FedericoLeiva12) [inline src/store/reducer/RootReducer.ts]: use destructuring
(state, {payload})

**Pull request merged**